includes/traits/llms-trait-earned-engagement-meta-box.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * LifterLMS Eearned Engagements (Certificate/Achievement) Meta Box trait.
+ *
+ * @package LifterLMS/Traits
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * LifterLMS Eearned Engagements (Certificate/Achievement) Meta Box trait.
+ *
+ * **This trait should only be used by classes that extend from the {@see LLMS_Admin_Metabox} class.**
```

Comment:
```suggestion
 * This trait should only be used by classes that extend from the {@see LLMS_Admin_Metabox} class.
```

Emphasis is unnecessary, I think.